src/shared/utils/rangeToInterval.test.ts

```diff
@@ -0,0 +1,159 @@
+import {
+  rangeToSQLInterval,
+  rangeToInfluxQLInterval,
+} from 'src/shared/utils/rangeToInterval'
+import {SELECTABLE_TIME_RANGES} from 'src/shared/constants/timeRanges'
+import {DurationTimeRange} from 'src/types'
+
+describe('rangeToSQLInterval:', () => {
```

Comment:
The tests under `rangeToSQLInterval` are copied from the old file `src/shared/utils/sqlInterval.test.ts` which is deleted/renamed in this PR.